examples/gmd_timings/Makefile

```diff
@@ -0,0 +1,37 @@
+DOCKER_IMAGE=gcr.io/vcm-ml/fv3gfs-wrapper:ams2021
```

Comment:
Will there be a `gmd` tagged image, or the like?  Or maybe there is no material difference?

Reply:
There have only been changes to the example code and not the wrapper itself since the ams image, so I'm thinking we'll leave it. In any case I will make a zenodo release of the image (I believe docker can be exported to zip).

Reply:
Sounds good.